validator/build.py

```diff
@@ -164,11 +164,11 @@ def GenValidatorProtoascii(out_dir):
   assert re.match(r'^[a-zA-Z_\-0-9]+$', out_dir), 'bad out_dir: %s' % out_dir
 
   protoascii_segments = [open('validator-main.protoascii').read()]
-  extensions = glob.glob('extensions/*/0.1/validator-*.protoascii')
+  extensions = glob.glob('extensions/*/(0.1|1.0)/validator-*.protoascii')
```

Comment:
I don't see what that would change since that file does not enumerate all versions. The protoascii should define which tag and extension is for which version.